src/main.js

```diff
@@ -72,12 +116,10 @@ function displayIdeaCards(){
 }
 
 function disableEnableButton(){
-  if (inputTitle.value !== '' && inputBody.value !== ''){
-    saveButton.disabled = false;
-    saveButton.classList.add('buttonEnabled');
-  } else {
+  if (!inputTitle.value.trim() || !inputBody.value.trim()){
```

Comment:
This was really clever! Great job for refactoring this to clear out spaces from being added and the card button accepting it as input.